src/d3a/d3a_core/d3a_jobs.py

```diff
@@ -44,6 +44,10 @@ def start(scenario, settings, events, aggregator_device_mapping):
     logging.getLogger().setLevel(logging.ERROR)
 
     scenario = decompress_and_decode_queued_strings(scenario)
+    logging.getLogger().error(f"Scenario details are : {0}: {scenario}")
+    logging.getLogger().error(f"Settings details are : {0}: {settings}")
+    logging.getLogger().error(f"Events details are : {0}: {events}")
+    logging.getLogger().error(f"Aggregator device mapping details are : {0}: {aggregator_device_mapping}")
```

Comment:
Travis is complaining about this line, that it is too long. I think that there is something wrong with the git precommit hooks in your local machine, otherwise you would not be able to commit this change.

As for the actual fix, if you split this line in 2 lines the problem will resolve.